package.json

```diff
@@ -2,19 +2,21 @@
   "name": "@netlify/zip-it-and-ship-it",
   "version": "4.23.2",
   "description": "Zip it and ship it",
-  "main": "./src/main.js",
+  "main": "./dist/main.js",
   "bin": {
-    "zip-it-and-ship-it": "src/bin.js"
+    "zip-it-and-ship-it": "dist/bin.js"
   },
   "files": [
-    "src/**/*.js"
+    "dist"
```

Comment:
Will `dist` only contain the transpiled JavaScript files? For example, if we have a non-TypeScript file (e.g. JSON) somewhere in `src`, will it end up in `dist` or only the files that `tsc` processes? I'm asking this because the current `files` glob purposely excludes any extraneous files from the npm package, and I'd like to keep that behaviour.

Reply:
`tsc` will replicate the structure in `src`, but only with regards to source files. AFAIK, a JSON file would be copied over, but only if it's imported from one of the TS files. But I'll verify that.

Reply:
If we enabled tsc's `resolveJsonModule` option, it would copy over directly-imported JSON files. Since we don't do that, it will not generate anything into `dist` except `.js`, `.d.ts` and `.js.map`.